src/mongo/s/stats/joom_latency_histogram.h

```diff
@@ -43,6 +71,16 @@ class JoomLatencyHistogram {
         uint64_t successCount = 0;
         uint64_t errorCount = 0;
         uint64_t nonUserCount = 0;
+
+        void _add(const HistogramData* data) {
+            for (size_t i = 0; i < kMaxBuckets; i++) {
+                this->buckets[i] += data->buckets[i];
+            }
+            this->sum += data->sum;
+            this->successCount += data->successCount;
+            this->errorCount += data->errorCount;
+            this->nonUserCount += nonUserCount;
```

Comment:
Missing `data->` on the right side of `+=`?

Reply:
Exactly, thanks! Fixed.